### PR TITLE
feat: refactor text executors pt 2

### PR DESCRIPTION
**Pull request by tadejsv**
This refactors

- [x] Tfidf
- [x] Spacy
- [x] TorchText
- [x] TorchSentence
- [x] Laser

I include the CI fix as well - seems to work well

### CODE REVIEW

File: jinahub/encoders/text/LaserEncoder/README.md [hunk 1]
```diff
@@ -25,17 +33,8 @@ Here is an example usage of the **LaserEncoder**.
 
 ```python
 from jina import Flow, Document
+
 f = Flow().add(uses='jinahub+docker://LaserEncoder')
 with f:
-    resp = f.post(on='foo', inputs=Document(text='hello Jina'), return_results=True)
+    resp = f.post(on='foo', inputs=Document(text='hello Jina'), on_done=print)
```
Comment (bwanglzu) [inline jinahub/encoders/text/LaserEncoder/README.md:39]: no return_results we can remove `resp`

**Review by bwanglzu — CHANGES_REQUESTED** [review]
minor thing

File: jinahub/encoders/text/LaserEncoder/README.md (hunk 1 above)
Comment (tadejsv) [reply jinahub/encoders/text/LaserEncoder/README.md]: done

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**